### ISSUE TITLE
backticks around whitespace behave differently than backticks around non-whitespace.

### ISSUE THREAD

**Issue by dkg** [cabo/kramdown-rfc2629#108]
Over in [draft-ietf-openpgp-crypto-refresh-02](https://datatracker.ietf.org/doc/html/draft-ietf-openpgp-crypto-refresh) ([source](https://gitlab.com/openpgp-wg/rfc4880bis/-/blob/main/crypto-refresh.md)) we have this source:

```
Dash-escaped cleartext is the ordinary cleartext where every line starting with a dash `-` (0x2D) is prefixed by the sequence dash `-` (0x2D) and space ` ` (0x20).
```

It renders (via `kramdown-rfc2629 --v3`) into the following:

```
Dash-escaped cleartext is the ordinary cleartext where every line starting with a dash <spanx style="verb">-</spanx> (0x2D) is prefixed by the sequence dash <spanx style="verb">-</spanx> (0x2D) and space ` ` (0x20).
```

this results in the infelicitous `.txt` version using xml2rfc 3.5.0:

```
   Dash-escaped cleartext is the ordinary cleartext where every line
   starting with a dash "-" (0x2D) is prefixed by the sequence dash "-"
   (0x2D) and space ` ` (0x20).
```

This was [noticed by Ángel in a series of nit-picks](https://mailarchive.ietf.org/arch/msg/openpgp/nMOv43axRuMzpEVKH3Ijr4-fkuk/).

**Comment by dkg** [comment]
Just to be clear: what's problematic is that an attempt to refer to the space character is different than the very similar attempt to refer to the dash character.

**Comment by cabo** [comment]
Yes.  How do you think should the HTML for a blank codespan look like?
This is clearly an upstream (kramdown) problem, but what should I tell Thomas how to handle this?

**Comment by cabo** [comment]
The commonmark spec https://spec.commonmark.org/0.29/#code-spans acknowledges that there are codespans that consist entirely of spaces.  But for now it seems you'll have to use the workaround <code> </code> for those.

**Comment by cabo** [comment]
Weird.  It works at the start of the paragraph.  Definitely bug territory.

**Comment by cabo** [comment]
Now https://github.com/gettalong/kramdown/issues/704

**Comment by cabo** [comment]
OK, here is the story.  Single backticks surrounded by whitespace don't turn into codespans.  So use double backticks (or any other repetition).   One leading and one trailing space are removed, though, so the official way to do a `<code> </code>` in kramdown is ```` ``   `` ```` (there are three spaces between the ``` `` ```; hard to see).

**Comment by cabo** [comment]
"Fixed" in 1.4.1 by picking up the upstream "fix" in kramdown 2.3.1.
